xcube/webapi/ows/stac/controllers.py

```diff
@@ -342,10 +353,12 @@ def _get_dataset_feature(ctx: DatasetsContext,
             ],
         },
         "properties": {
-            "xcube:dimensions": dict(dataset.dims),
-            "xcube:variables": variables,
-            "xcube:coordinates": coordinates,
-            "xcube:attributes": dict(dataset.attrs),
+            "datacube:dimensions": dc_dimensions,
+            "datacube:variables": dc_variables,
```

Comment:
Why `datacube` and not `cube`? This won't validate against the datacube schema.

Reply:
Argh, I overlooked the most obvious thing! Will change...